src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -96,6 +98,21 @@ testRule(rule, {
   }, {
     code: "a { font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', sans-serif; }",
     message: messages.rejected("BlinkMacSystemFont"),
+  }, {
+    code: "a { font: italic 300 16px/30px Arial, serif; }",
+    message: messages.expected("Arial"),
+    line: 1,
+    column: 32,
+  }, {
```

Comment:
> I think unknown numerical font weights get ignored, like other invalid CSS values.

Yep, I just checked that this happens.

Either way, is this fix ready to go in? Using `isNumbery` seems to allow the accurate finding of the font family name irrespective of whether the font weight is valid or not. As the purpose of this rule is to check the quotes around the font family name, this seems good to me.